### ISSUE TITLE
Info missing on docs or on the installer?

### ISSUE THREAD

**Issue by alfetopito** [progrium/dokku#985]
Hey there!

I managed to get dokku working on vagrant locally, now I'm trying to boot it on aws.

Following the [instructions](http://progrium.viewdocs.io/dokku/installation) -- basically running twice this

``` bash
wget -qO- https://raw.github.com/progrium/dokku/v0.3.15/bootstrap.sh | sudo DOKKU_TAG=v0.3.15 bash
```
 It ends with

``` bash
...
dokku plugins-install
git describe --tags > ~dokku/VERSION  2> /dev/null || echo '~master (2015-02-18T13:11+0000)' > ~dokku/VERSION

Almost done! For next steps on configuration:
  http://progrium.viewdocs.io/dokku/installation#user-content-configuring
```

The tag (#user-content-configuring) leads nowhere.

Alright, that is not a big deal.
The real problem is I can't install the webinstaller. When I run `cd /root/dokku && make dokku-installer`, first nothing happens because `/root/dokku/` doesn't exist.
So I just `cd dokku` the current folder I'm in, ending up at `/home/ubuntu/dokku` and run the previous command, and I get this:

``` bash
$ sudo make dokku-installer
apt-get install -qq -y ruby
test -f /var/lib/dokku/.dokku-installer-created || gem install rack -v 1.5.2 --no-rdoc --no-ri
Successfully installed rack-1.5.2
1 gem installed
test -f /var/lib/dokku/.dokku-installer-created || gem install rack-protection -v 1.5.3 --no-rdoc --no-ri
Successfully installed rack-protection-1.5.3
1 gem installed
test -f /var/lib/dokku/.dokku-installer-created || gem install sinatra -v 1.4.5 --no-rdoc --no-ri
Successfully installed sinatra-1.4.5
1 gem installed
test -f /var/lib/dokku/.dokku-installer-created || gem install tilt -v 1.4.1 --no-rdoc --no-ri
Successfully installed tilt-1.4.1
1 gem installed
test -f /var/lib/dokku/.dokku-installer-created || ruby /root/dokku/contrib/dokku-installer.rb onboot
ruby: No such file or directory -- /root/dokku/contrib/dokku-installer.rb (LoadError)
make: *** [dokku-installer] Error 1
```

It seems it really wants the dokku folder to be at `/root/dokku`, so I run the installer again, this time as root, at `/root`, then run `make dokku-installer`, and voilà, worked!

So, did I miss anything where it says I should install the stuff as root rather than sudo? Or should the installer take care of that?

Anyway, it worked for me this way, just a heads up in case it shouldn't be like that.

**Comment by simenbrekken** [comment]
I had the exact same experience where I had to `sudo su` and be in `/root` for the installer to work. After that I got the web installer up and working but it came without any instructions (e.g what am I supposed to use the "Admin Access Public Key" for?)

I've added all my output logs here, starting with a fresh ubuntu-trusty-14.04-amd64-server image:
https://gist.github.com/simenbrekken/154c4e8efe7a7e42be32

**Comment by josegonzalez** [comment]
@alfetopito Odd that it didn't create that directory, I'll try running on AWS and seeing what's going on. In the future, we'll be promoting a debian-package based installation (which is how we build DO images now) but it's important we get the bootstrap script correct :)

@simenbrekken what do you mean by "Admin Access Public Key"?

**Comment by simenbrekken** [comment]
@josegonzalez That there's a text box titled "Admin Access - "Public Key" but it's never mentioned anywhere in the docs what I'm supposed to do with it.

**Comment by alfetopito** [comment]
@simenbrekken That is the public key you will use to access your dokku instance, to be able to push-to-deploy like: `git push dokku master`, after having added the remote to you git repo with:
`git remote add dokku dokku@<your dokku instance ip or hostname>:<yourapp>`

So you need to paste your ssh public key, the one you use with that git repository  —  generally found at `~/.ssh/id_rsa.pub`

**Comment by simenbrekken** [comment]
@alfetopito So the key that's already in the text box is just a pre-generated public key?

**Comment by alfetopito** [comment]
@simenbrekken No. I think the key is the default of the system you installed Dokku on.
I don't think it generates a key for you. Besides, that wouldn't be useful.
Anyway, I believe @josegonzalez would be able to answer it with more accuracy.

**Comment by josegonzalez** [comment]
The key there is the one used to allow pushing code to the dokku instance (you can add more via ssh later). If you bring it up from a pre-made image on Digital Ocean and specify that the droplet be brought up with an ssh key, it is automatically inserted into that text box. Otherwise, the text box will be blank.

Note that it's contents are sourced from `/root/.ssh/id_rsa.pub`, so if you create a new image on a cloud provider and have an ssh key there, it *will* be loaded.

**Comment by DavidLGoldberg** [comment]
I had to sudo su; cd /root as well.  Might want to add that AWS condition to the tutorial?

**Comment by craigbeck** [comment]
Issues similar to https://github.com/progrium/dokku/issues/1016 where the script documented for install does not work as described.

+1 for adding to documentation (or installer) info about what the key is (in cases where there is one already) or what it should be (for cases where the input box is empty). I'm not sure if I'm entirely clear what I shoul dbe putting in there.

**Comment by rorykoehler** [comment]
In case you need it the correct link for further configuration documentation is http://progrium.viewdocs.io/dokku/advanced-installation#user-content-configuring

This is for the broken link referenced here: 

```
Almost done! For next steps on configuration:
  http://progrium.viewdocs.io/dokku/installation#user-content-configuring
```

I have already updated it and made a [pull request](https://github.com/progrium/dokku/pull/1079).

**Comment by josegonzalez** [comment]
Closed by #1058.
